### PR TITLE
Resolve the latest cookstyle warnings

### PR DESCRIPTION
**Pull request by Xorima**
Latest cookstyle changes

Signed-off-by: Jason Field <jason@avon-lea.co.uk>

# Description

[Describe what this change achieves]

## Issues Resolved

[List any existing issues this PR resolves]

## Check List

- [ ] All tests pass. See https://github.com/sous-chefs/apache2/blob/master/TESTING.md
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable

### CODE REVIEW

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please include a CHANGELOG entry.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: metadata.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@
 chef_version     '>= 13.9'
 license          'Apache-2.0'
 description      'Installs and configures apache2'
-long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
+
```
Comment (tas50) [inline metadata.rb]: might as well just nuke this line

File: resources/mod_ssl.rb [hunk 2]
```diff
@@ -60,7 +60,7 @@
     with_run_context :root do
       package 'mod_ssl' do
         notifies :run, 'execute[generate-module-list]', :immediately
-        only_if { platform_family?('rhel', 'fedora', 'suse', 'amazon') }
```
Comment (tas50) [inline resources/mod_ssl.rb]: ?

Comment (Xorima) [reply resources/mod_ssl.rb]: Should be just removing suse, suse doesn't have a mod_ssl ...

**Comment by Xorima** [comment]
@tas50 / @damacus  can I get an approval on this?

File: libraries/helpers.rb [hunk 3]
```diff
@@ -195,6 +195,8 @@ def default_docroot_dir
           '/usr/local/www/apache24/data'
         when 'suse'
           '/srv/www/htdocs'
+        when 'opensuseleap'
```
Comment (tas50) [inline libraries/helpers.rb]: you could put this on a single line with when 'suse, 'opensuseleap'

**Pull request merged**